### PR TITLE
Modifies Stinger receipe, taste and description for Irish Car Bomb, Green Beer, Goldschläger, Blood Mead, Mead, and Grape Soda 

### PR DESCRIPTION
**Pull request by RiskySikh**
## About The Pull Request

It modifies the recipe for the Stinger, originally it was 10 whiskey and 5 creme de menthe; however, in real life the actual recipe it was 10 cognac and 5 creme de menthe. I presume it was whiskey because cognac wasn't available during that time, but now whiskey is replaced with cognac. As well, I fixed some grammatical issues, changed the description of the drinks, and the taste. To make it more funnier, and unique in a way. 

## Why It's Good For The Game

Makes the receipt for Stinger realistic and instead of 10 units of whiskey is instead 10 units of cognac. Makes the description and taste for drinks mentioned above more "humorous", well in my way. I am not sure for you fellas. In general, it's just some minor fixes and modifications for bartender.

## Changelog
:cl:
tweak: Modified the recipe for Stinger instead of 10 units of Whiskey its now 10 units of Cognac. Modified the description and taste for Irish Car Bomb, Green Beer, Goldschlager, Blood Mead, Mead, and Grape Soda. 
/:cl:

### CODE REVIEW

**Comment by ATH1909** [comment]
The title should say "modifies" instead of "fixes"; it's misleading.

**Comment by RiskySikh** [comment]
> 
> 
> The title should say "modifies" instead of "fixes"; it's misleading.

fixed it sorry

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -1090,14 +1090,14 @@ All effects don't start immediately, but rather get worse over time; the rate is
 
 /datum/reagent/consumable/ethanol/irishcarbomb
 	name = "Irish Car Bomb"
-	description = "Mmm, tastes like chocolate cake..."
+	description = "Mmm, tastes like Anger."
 	color = "#2E6671" // rgb: 46, 102, 113
 	boozepwr = 25
 	quality = DRINK_GOOD
-	taste_description = "delicious anger"
+	taste_description = "the spirit of Ireland"
```
Comment (Fikou) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: you replaced the description to tastes like anger but you removed it tasting like anger

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 2]
```diff
@@ -100,10 +100,10 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	name = "Green Beer"
 	description = "An alcoholic beverage brewed since ancient times on Old Earth. This variety is dyed a festive green."
 	color = "#A8E61D"
-	taste_description = "green piss water"
+	taste_description = "crayons and piss"
```
Comment (Qustinnus) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: the entire joke is that some beers gets called green piss water, why rename this to include crayons? ruins the joke.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 3]
```diff
@@ -100,10 +100,10 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
 	glass_icon_state = "greenbeerglass"
 	glass_name = "glass of green beer"
-	glass_desc = "A freezing pint of green beer. Festive."
+	glass_desc = "Mmm... Grayons..."
```
Comment (Qustinnus) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: same here

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm (hunk 2 above)
Comment (RiskySikh) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: > 
> 
> the entire joke is that some beers gets called green piss water, why rename this to include crayons? ruins the joke.

my bad man I fixed it to it's original state

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm (hunk 3 above)
Comment (RiskySikh) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: fixed it as well

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm (hunk 1 above)
Comment (RiskySikh) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: > 
> 
> you replaced the description to tastes like anger but you removed it tasting like anger

my bad, I fixed it

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 4]
```diff
@@ -430,11 +430,11 @@ All effects don't start immediately, but rather get worse over time; the rate is
 
 /datum/reagent/consumable/ethanol/goldschlager
 	name = "Goldschlager"
-	description = "100 proof cinnamon schnapps, made for alcoholic teen girls on spring break."
+	description = "100% real cinnamon schnapps, made for alcoholic teen girls on spring break."
```
Comment (kriskog) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: Why this? Proof describes alcohol content.

Comment (RiskySikh) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: > 
> 
> Why this? Proof describes alcohol content.

Understood, I will fix the changes.

Comment (kriskog) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: I dont think you understood. 100 proof is how americans would write 50% alcohol content.
https://en.wikipedia.org/wiki/Alcohol_proof

Comment (RiskySikh) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: So different ways of describing the alcohol content, gotcha.

Comment (kriskog) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: Right. And the current text is
```
description = "100% proof cinnamon schnapps, made for alcoholic teen girls on spring break."
```

Comment (RiskySikh) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: Aye

Comment (RiskySikh) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: > 
> 
> Right. And the current text is
> 
> ```
> description = "100% proof cinnamon schnapps, made for alcoholic teen girls on spring break."
> ```

Now I understand the issue, I reverted the changes of the description for goldschlager. It's supposed to be 100 proof not 100% proof.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 5]
```diff
@@ -434,7 +434,7 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	color = "#FFFF91" // rgb: 255, 255, 145
 	boozepwr = 25
 	quality = DRINK_VERYGOOD
-	taste_description = "burning cinnamon"
+	taste_description = "burning cinnamon and tears of drunk teen girls"
```
Comment (Rohesie) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: Is this really necessary?

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 6]
```diff
@@ -1090,14 +1090,14 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	description = "Mmm, tastes like the free Irish state."
 	color = "#2E6671" // rgb: 46, 102, 113
 	boozepwr = 25
 	quality = DRINK_GOOD
-	taste_description = "delicious anger"
+	taste_description = "the spirit of Ireland"
 	glass_icon_state = "irishcarbomb"
 	glass_name = "Irish Car Bomb"
-	glass_desc = "An Irish car bomb."
+	glass_desc = "Britain's worst nightmare."
```
Comment (Rohesie) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: Is this really necessary?

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm (hunk 5 above)
Comment (RiskySikh) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: fixed it

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm (hunk 6 above)
Comment (RiskySikh) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: fixed it

**Pull request merged**